Blockzilla/AppDelegate.swift

```diff
@@ -50,10 +50,21 @@ class AppDelegate: UIResponder, UIApplicationDelegate, ModalDelegate, AppSplashC
         setupErrorTracking()
         setupTelemetry()
         TPStatsBlocklistChecker.shared.startup()
+        browserViewController.tipManager = TipManager()
```

Comment:
Lets flip this around a bit and have `TipManager` manage the singleton. I think the changes would be:
* Add `static let shared = TipManager()` in the `TipManager` class
* Modify `BrowserViewController` constructor to include the argument `tipmanager: TipManager = TipManager.shared`
* Modify `HomeView` constructor to also take in a TipManager and optionally pass that in based on if tips are visible at that time.